Topic-1-HTML/exercise-1.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+        <meta charset="utf-8">
+</head>    
+
```

Comment:
Please add a description in the pull request to let know the reviewers what is going on.

Something like:

**Tasks**

- [x] First exercise
- [x] Second exercise
- [ ] Third exercise